### PR TITLE
Post SDK cleanup

### PR DESCRIPTION
**Pull request by jaredpar**
Couple of post SDK clean up items:

- Move samples to new SDK
- Remove ProjectTypeGuids and TargetFrameworkProfile from new SDK projects
- Enforce the above with BuildBoss

### CODE REVIEW

**Comment by Pilchie** [comment]
How long until we start using globbing?

**Comment by jaredpar** [comment]
@Pilchie 

> How long until we start using globbing?

@srivatsn mentioned in another thread he has a tool to convert projects to use globbing. Wanted to test it on Roslyn. So basically whenever he runs the tool is when we get globbing.

**Comment by jaredpar** [comment]
I was actually planning on converting to globbing next week. But @srivatsn offered so letting him have a go at it.

This way I also don't get blamed for any issues it causes 😄

**Comment by sharwell** [comment]
@jaredpar It's actually #20568 and #20569 that affected this. The former has been merged, which means this PR can be updated to include the missing **launchSettings.json** files for the sample projects and remove the unnecessary lines in the build targets.

**Comment by jaredpar** [comment]
retest windows_release_vs-integration_prtest  please

**Comment by jaredpar** [comment]
retest windows_debug_vs-integration_prtest please

File: src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj [hunk 1]
```diff
@@ -12,7 +12,6 @@
     <AssemblyName>Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests</AssemblyName>
     <TargetExt>.dll</TargetExt>
     <TargetFramework>netstandard1.3</TargetFramework>
-    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```
Comment (tannergooding) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: You should add `<ProjectCapability Include="TestContainer" />` to all the Unit Test projects.

Comment (srivatsn) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: It's UnitTestContainer

Comment (jaredpar) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: What does that do though?

Comment (sharwell) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: @tannergooding It's in most already: https://github.com/dotnet/roslyn/blob/master/build/Targets/Imports.targets#L45-L48

Comment (sharwell) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: @jaredpar It should eventually change the icon of the projects in Solution Explorer, but I was never told where to file a bug for that: https://github.com/dotnet/roslyn/pull/18363#issuecomment-299922406

In the meantime, it's used by LUT to filter out projects that can't have tests.

Comment (srivatsn) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: File the bug in https://gitbub.com/Microsoft/vstest

Comment (tannergooding) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: In short, it is meant to light up functionality meant for projects which contain unit tests.

Comment (jaredpar) [inline src/CodeStyle/CSharp/Tests/CSharpCodeStyleTests.csproj]: Gotcha. Let me grab that in my next PR that I'm working on. This finally went full green.

**Pull request merged**